packages/js/src/settings/routes/site-representation.js

```diff
@@ -42,7 +42,7 @@ const SiteRepresentation = () => {
 	const isPremium = useSelectSettings( "selectPreference", [], "isPremium" );
 	const premiumUpsellConfig = useSelectSettings( "selectUpsellSettingsAsProps" );
 	const mastodonPremiumLink = useSelectSettings( "selectLink", [], "https://yoa.st/get-mastodon-integration" );
-	const mastodonUrlLink = useSelectSettings( "selectLink", [], "https://yoa.st/mastodon-shortlink" );
+	const mastodonUrlLink = useSelectSettings( "selectLink", [], "http://yoa.st/site-representation-mastodon" );
```

Comment:
Is there a reason to use http here instead of https link the one above. If there is we might want to change the other one as well :)

Reply:
```suggestion
	const mastodonUrlLink = useSelectSettings( "selectLink", [], "https://yoa.st/site-representation-mastodon" );
```